### ISSUE TITLE
Alt attribute

### ISSUE THREAD

**Issue by jim-rosales** [markAnthonyDay/prj-rev-bwfs-dasmoto#3]
When adding images to a site, its best practice to include the [alt attribute](https://en.wikipedia.org/wiki/Alt_attribute) as well. The alt attribute optimizes your site for search engines, is used for [accessibility](https://www.codecademy.com/articles/accessibility), and as a backup, in case the image cannot be rendered. 

For example:
`<img src="https://s3.amazonaws.com/codecademy-content/courses/freelance-1/unit-2/hacksaw.jpeg" alt="Hacksaw">`

https://github.com/markAnthonyDay/prj-rev-bwfs-dasmoto/blob/3e386065759778463b7bc6591d44f0f3c09a4c63/Dasmotos/index.html#L23
